modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -403,6 +403,34 @@ TEST(TestAgeGenderIE, GenericInfer)
     normAssert(cv::gapi::ie::util::to_ocv(ie_gender), gapi_gender, "Test gender output");
 }
 
+TEST(TestAgeGenderIE, InvalidConfig)
+{
+    initDLDTDataPath();
+
+    std::string model_path   = findDataFile(SUBDIR + "age-gender-recognition-retail-0013.xml");
+    std::string weights_path = findDataFile(SUBDIR + "age-gender-recognition-retail-0013.bin");
+    std::string device_id    = "CPU";
+
+    // Configure & run G-API
+    cv::GMat in;
+    GInferInputs inputs;
+    inputs["data"] = in;
+
+    auto outputs = cv::gapi::infer<cv::gapi::Generic>("age-gender-generic", inputs);
+    auto age     = outputs.at("age_conv3");
+    auto gender  = outputs.at("prob");
+    cv::GComputation comp(cv::GIn(in), cv::GOut(age, gender));
+
+    cv::gapi::ie::Params<cv::gapi::Generic> pp{"age-gender-generic",
+                                                model_path,
+                                                weights_path,
+                                                device_id,
+                                                {{"unsupported_config", "some_value"}}};
```

Comment:
No. Putting everything to a cosntructor bloats up everything.

I'd expect to see
```c++
pp.cfgPlugin(map);
```
or even
```c++
pp
    .cfgPlugin("key", "value")
    .cfgPlugin("key", "value")
```

Reply:
Fixed

Reply:
I propose to change the method name to `setCfg` or something similar. `cfgPlugin` looks like it expects some plugin name at input and may be misleading. In its turn `setCfg` can conflict with `cfgInputs/Outputs` though since there are different meanings of `cfg` in this names

Reply:
we have all other methods starting with `cfg`, and via this you configure plugin, though I agree it doesn't look very good

Reply:
hmm... except `constInput()`. :)

Maybe `.pluginConfig` is what we need - but it can always be added atop.

Reply:
`setPluginConfig()`  and `getPluginConfig()` (optional, but can be used in tests)

Reply:
Ok, rename to `pluginConfig`